### ISSUE TITLE
How was the validation split derived?

### ISSUE THREAD

**Issue by tonyduan** [locuslab/robust_overfitting#1]
Great work and thanks for the repo!

I'm seeing this line in the code to define the validation set:
```
dataset = torch.load("cifar10_validation_split.pth")
```

I couldn't find this file in the codebase. Would you mind pointing me to the file, or explaining how the split of 1,000 images was defined?

Thanks,
Tony

**Comment by riceric22** [comment]
Hi Tony, 

It's just a random subset, chosen via a random permutation of the dataset. I seem to have forgotten to commit it to the repo, thanks for pointing this out! The actual file itself is a copy of the CIFAR10 dataset, but you can generate it with the `generate_validation.py` script that I just pushed with 19a6740cc84057058029e67cd9518fc2346813de which should produce the same split since it uses a fixed seed. 

Since the split was random, I wouldn't expect the performance to change drastically if you just randomly split it yourself. In fact, I would now recommend that you just use `torch.utils.data.random_split(dataset, lengths)` instead of generating it separately with the script (which I didn't know about when I first wrote this code), it's cleaner and simpler. 

~Eric
